docs/observability/aws/integrations/amazon-sqs.md

```diff
@@ -0,0 +1,169 @@
+         "awsRegion":"us-east-1",
+         "sourceIPAddress":"195.186.216.125",
+         "userAgent":"AWS Internal",
+         "requestParameters":{
+           "attributes":{
+        "Policy":"{\"Version\":\"2008-10-17\",\"Id\":\"__default_policy_ID\",\"Statement\":[{\"Sid\":\"__owner_statement\",\"Effect\":\"Allow\",\"Principal\":{\"AWS\":\"arn:aws:iam::956882708938:root\"},\"Action\":\"SQS:*\",\"Resource\":\"arn:aws:sqs:us-east-1:956882708938:JayanatTest4\"},{\"Sid\":\"topic-subscription-arn:aws:sns:us-east-1:956882708938:SNSAppSomya01\",\"Effect\":\"Allow\",\"Principal\":{\"AWS\":\"*\"},\"Action\":\"SQS:SendMessage\",\"Resource\":\"arn:aws:sqs:us-east-1:956882708938:JayanatTest4\",\"Condition\":{\"ArnLike\":{\"aws:SourceArn\":\"arn:aws:sns:us-east-1:956882708938:SNSAppSomya01\"}}}]}"
+      },
+      "queueUrl":"https://sqs.us-east-1.amazonaws.com/123456789033/pull_private_submodule_jobs.fifo"
+         },
+         "responseElements":null,
+         "requestID":"635ae9dd-83cc-5b42-890e-b273c168cb35",
+         "eventID":"874213f3-d852-481d-bfd1-677c20f97427",
+         "readOnly":false,
+         "eventType":"AwsApiCall",
+         "managementEvent":true,
+         "recipientAccountId":"123456789033",
+         "eventCategory":"Management",
+         "sessionCredentialFromConsole":"true"         
+}
+```
+
+
+
+### Sample Query
+
+**Messages Received (Metric based)**:
+
+```
+metric=NumberOfMessagesReceived Statistic=Sum account=* region=* namespace=* queuename=* | sum by account, region, namespace, queuename
+```
+
+**Top 10 users (CloudTrail Log based)**:
+
+```sql
+account=* region=* namespace=aws/sqs eventname eventsource "sqs.amazonaws.com"
+| json "userIdentity", "eventSource", "eventName", "awsRegion", "recipientAccountId", "requestParameters", "responseElements", "sourceIPAddress","errorCode", "errorMessage" as userIdentity, event_source, event_name, region, recipient_account_id, requestParameters, responseElements, src_ip, error_code, error_message nodrop
+| json field=userIdentity "accountId", "type", "arn", "userName" as accountid, type, arn, username nodrop
+| json field=requestParameters "queueUrl" as queueUrlReq nodrop
+| json field=responseElements "queueUrl" as queueUrlRes nodrop
+| where event_source="sqs.amazonaws.com"
+| if(event_name="CreateQueue", queueUrlRes, queueUrlReq) as queueUrl
+| parse regex field=queueUrl "(?<queueName>[^\/]*$)"
+| where (tolowercase(queuename) matches tolowercase("*")) or isBlank(queuename)
+| if (isBlank(recipient_account_id), accountid, recipient_account_id) as accountid
+| if (isEmpty(error_code), "Success", "Failure") as event_status
+| count as event_count by username
+| top 10 username by event_count, username asc
+```
+
+## Field in Field Schema
+
+Login to Sumo Logic,  go to Manage Data > Logs > Fields. Search for the “queuename” field. If not present, create it. Learn how to create and manage fields [here](https://help.sumologic.com/Manage/Fields#manage-fields).
```

Comment:
Login to Sumo Logic, go to **Manage Data** > **Logs** > **Fields**. Search for the “queuename” field. If not present, create it. Learn how to create and manage fields [here](/docs/manage/fields/#manage-fields).